### ISSUE TITLE
[member-delimiter-style] Default rule not enforced

### ISSUE THREAD

**Issue by richardsimko** [typescript-eslint/typescript-eslint#2159]
**Repro**

https://github.com/richardsimko/eslint-member-delimiter-style-test

```JS
module.exports = {
  'env': {
    'node': true
  },
  'extends': [
    'eslint:recommended',
    'plugin:@typescript-eslint/eslint-recommended'
  ],
  'parser': '@typescript-eslint/parser',
  'parserOptions': {
    'ecmaVersion': 2018
  },
  'plugins': [
    '@typescript-eslint'
  ]
};
```

```TS
interface Test {
  member1: string
  member2: number;
}
```

**Expected Result**
According to the docs on `member-delimiter-style` it should error on the missing semi colon.

**Actual Result**
No error.

**Additional Info**
If I explicitly define the rule as an error with the config mentioned as default in the docs it works as expected.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `3.1.0` |
| `@typescript-eslint/parser`        | `3.1.0` |
| `TypeScript`                       | `3.9.3` |
| `ESLint`                           | `7.1.0` |
| `node`                             | `12.16.2` |
| `npm`                              | `n/a` |
| `yarn`                              | `1.22.4` |

**Comment by richardsimko** [comment]
This is a regression, it works with the following versions:

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `3.0.1` |
| `@typescript-eslint/parser`        | `3.0.1` |

**Comment by bradzacher** [comment]
`member-delimiter-style` is not a recommended rule.
You must turn it on yourself in your config if you want to use it.

**Comment by richardsimko** [comment]
Thanks for the reply. Did this change since 3.0.1 or was it just a bug that it was enabled by default previously?

**Comment by bradzacher** [comment]
This rule was removed from the recommended set in the v3 release.

**Comment by richardsimko** [comment]
Thanks, that explains everything! :)
